data_structures/queue/fifo_animal_shelter.py

```diff
@@ -17,9 +50,9 @@ def __repr__(self):
 
     def enqueue(self, animal=None):
         """Insert an animal to the back of the queue"""
-        node = Node(animal)
+        node = Node(animal.species)
```

Comment:
You should be enqueueing the entire animal object instead of just its species. This strips off all other information about the animal which would be useful to have.